### DISCUSSION TITLE
Some folders in C:\Users\{name}\AppData\Roaming\OpenShell\Pinned don't show in Open-Shell

### DISCUSSION THREAD

**Discussion by coffent** [Open-Shell/Open-Shell-Menu#1903] [Q&A] [answered]
I have a number of folders in C:\Users\{name}\AppData\Roaming\OpenShell\Pinned.  Three such folders no longer appear in my Open-Shell listing, even though they are located in the above subfolder along with others that *do* appear, and even though they *used* to appear.  Do the folders need to be located somewhere else?  How can I get them to appear?

**Comment by ge0rdi** [comment, marked as answer]
How many folders do you have there?
I have created 15 empty folders in `Pinned` folder and I can see all of them in start menu (Windows7 style).

**Reply by coffent** [reply]
Thanks for the reply.  I think that's the problem.  I had 25 subfolders in the Pinned folder, but only 20 appear in Open-Shell.  When I reduced the number of folders to 20, then added 1 more, the new folder appeared but it bumped out another one.  I wasn't aware that there was a limit on the number of folders.  Is there a way of increasing the limit?

**Reply by ge0rdi** [reply]
Hmm, it seems that the menu will show as many items as it fits to the screen (single column). At least I haven't found any setting that should limit the height.

In my case I can have even 30 folders in the menu. And when using `Small icons` in skin settings I can even fit 53 of them.

**Comment by coffent** [comment]
I use Open Shell because I like it much better than the Windows version, though I make no claim to understanding how it works.
My Open Shell listing is divided into two parts:  an upper part that shows shortcuts contained in C:\Users\{name}\AppData\Roaming\OpenShell, and a lower part showing shortcuts contained in C:\Users\{name}\AppData\Roaming\OpenShell\Pinned.  Adding the two lists together I have 42 folders + shortcuts.  So I guess the limit for me is 42 total, which I can distribute among the pinned & non-pinned listings.
